### PR TITLE
feat: log the count of scenarios found

### PR DESCRIPTION
**Pull request by dirgim**
* Added logging to help debugging in staging

Replaces #32 because of refactoring changes

Signed-off-by: dirgim <kpavic@redhat.com>

### CODE REVIEW

File: controllers/snapshot/snapshot_adapter.go [hunk 1]
```diff
@@ -72,6 +72,10 @@ func (a *Adapter) EnsureAllIntegrationTestPipelinesExist() (results.OperationRes
 			"Application.Namespace", a.application.Namespace)
 	}
 
+	a.logger.Info("Found IntegrationTestScenarios for application",
```
Comment (srivickynesh) [inline controllers/snapshot/snapshot_adapter.go:75]: Also good to add this within https://github.com/redhat-appstudio/integration-service/blob/main/helpers/integration.go#L58.

Comment (dirgim) [reply controllers/snapshot/snapshot_adapter.go:75]: Yes, I think it's better to handle the logging here, within the `snapshot_adapter.go`, as we're able to tailor the message to the use-case.

**Review by srivickynesh — APPROVED** [review]
LGTM

**Review by hongweiliu17 — APPROVED** [review]
LGTM

**Pull request merged**